benchmarks/src/main/java/io/druid/benchmark/query/GroupByBenchmark.java

```diff
@@ -190,7 +190,47 @@ private void setupQueries()
       basicQueries.put("A", queryA);
     }
 
+    Map<String, GroupByQuery> nestedQueries = new LinkedHashMap<>();
+    { // nested.A
+      QuerySegmentSpec intervalSpec = new MultipleIntervalSegmentSpec(Arrays.asList(basicSchema.getDataInterval()));
+      List<AggregatorFactory> queryAggs = new ArrayList<>();
+      queryAggs.add(new LongSumAggregatorFactory(
+          "sumLongSequential",
+          "sumLongSequential"
+      ));
+
+      GroupByQuery subqueryA = GroupByQuery
+          .builder()
+          .setDataSource("blah")
+          .setQuerySegmentSpec(intervalSpec)
+          .setDimensions(Lists.<DimensionSpec>newArrayList(
+              new DefaultDimensionSpec("dimSequential", null),
+              new DefaultDimensionSpec("dimZipf", null)
+          ))
+          .setAggregatorSpecs(
+              queryAggs
+          )
+          .setGranularity(QueryGranularities.DAY)
+          .build();
+
+      GroupByQuery queryA = GroupByQuery
+          .builder()
+          .setDataSource(subqueryA)
+          .setQuerySegmentSpec(intervalSpec)
+          .setDimensions(Lists.<DimensionSpec>newArrayList(
+              new DefaultDimensionSpec("dimSequential", null)
+          ))
+          .setAggregatorSpecs(
+              queryAggs
+          )
+          .setGranularity(QueryGranularities.WEEK)
+          .build();
+
+      nestedQueries.put("A", queryA);
```

Comment:
The current way result in Param({"basic.A", "nested.A"}), if nested is put into basicQueries, it would be  Param({"basic.A", "basic.nested"}). Which structure do you think is more reasonable?